tsconfig.json

```diff
@@ -9,5 +9,5 @@
             "*": ["node_modules/*"]
         }
     },
-    "include": ["src/**/*"]
+    "include": ["src/main/**/*"]
```

Comment:
I was thinking we could move this `tsconfig` to main directory as it only is specific to it

Reply:
I think we also need to compile files in `src/*` directory, so added that.